### ISSUE TITLE
Create Dockerfile for production build of PWA

### ISSUE THREAD

**Issue by sharkySharks** [magento-research/pwa-studio#940]
Need to have a PWA production instance for demo purposes. The first step in this process includes creating a docker image that serves PWA from a production server. The current docker setup uses the development server for serving the application, ie `webpack-dev-server`, and cannot be used to run the application as the magento instance, https://m230-pwa-ent-0.testsonfire.com/, is set to `production` mode and therefore conflicts with local development and query verification mode. 

A proposal for a production build is to utilize the `node server.js` command to run the application with node and proxy it through nginx or another layer in order to handle web traffic and add additional monitoring of at least a `/healthz` endpoint.

This will use the upward-js module; a separate effort will be needed to serve the application using upward-php in the future.

**Comment by sharkySharks** [comment]
This work is complete but will ship together with #587 to see full end to end functionality.

**Comment by sharkySharks** [comment]
Work completed, adding hold label - waiting on #859 to be merged.

**Comment by sharkySharks** [comment]
nginx layer is not necessary at this moment as the "production" build is for demo purposes only and will not be handling a lot of web traffic. Health monitoring alerts can be turned on in Elastic Beanstalk.

**Comment by sharkySharks** [comment]
Removing HOLD after #859 merge.

**Comment by sharkySharks** [comment]
Adding `HOLD` label, waiting on #1012 to create pull request.

**Comment by sharkySharks** [comment]
PR: #1087
